### PR TITLE
[12.0][MIG] purchase_date_planned_manual

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Standard Migration

- [x]  purchase_stock dependency added (not sure if it is correct)

- [x] Add https://github.com/OCA/purchase-workflow/pull/707

CC @Eficent

### CODE REVIEW

File: purchase_date_planned_manual/tests/test_date_planned_manual.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+class TestDatePlannedManual(TransactionCase):
+
+    def setUp(self):
+        super(TestDatePlannedManual, self).setUp()
+        self.po_model = self.env['purchase.order']
+        self.pol_model = self.env['purchase.order.line']
+        self.pp_model = self.env['product.product']
+        # Create some partner, products and supplier info:
+        self.route_buy_id = self.env.ref('purchase_stock.route_warehouse0_buy')
+        self.rule = first(self.route_buy_id.pull_ids)
```
Comment (MiquelRForgeFlow) [inline purchase_date_planned_manual/tests/test_date_planned_manual.py]:
```suggestion
        self.rule = first(self.route_buy_id.rule_ids)
```

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by AdriaGForgeFlow — APPROVED** [review]
Functional test and code review :+1:

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request merged**